src/adapters/github.ts

```diff
@@ -90,11 +83,12 @@ class GithubAdapter extends GitAdapter {
           fullNamePath = 'repository.full_name'
       }
 
-      const searchResults = await paginateSearch(this.octokit, searchMethod, onRetry)({
-        q: search_query,
-      });
-
-      repoNames = searchResults.map((r: any) => _.get(r, fullNamePath) ).sort();
+      const searchResults: string[] = await this.octokit.paginate(
+        searchMethod,
+        { q: search_query },
+        (r: any) => r.items
```

Comment:
```suggestion
        (r: any) => r.data
```

Reply:
Are we missing a unit test for this?

Reply:
This is intentional. Per the [octokit pagination doc](https://octokit.github.io/rest.js/v18#pagination):

> octokit.paginate() accepts the same options as octokit.request(). You can optionally pass an additional function to map the results from each response. The map must return a new value, usually an array with mapped data.

> Note: the map function is called with the { data, headers, status, url } response object. The data property is guaranteed to be an array of the result items, even for list endpoints that respond with an object instead of an array, such as the search endpoints.

In this case, I'm passing in that optional function to map results which is called with `{ data, headers, status, url }`.

Reply:
Oops, I thought I had `.data`. Yes it should be data, resolving.